theme_kea/views/snippets/s_color_blocks_2.xml

```diff
@@ -10,27 +10,43 @@
     <xpath expr="//h2" position="before">
         <div class="o_we_bg_filter bg-black-50"></div>
     </xpath>
+    <xpath expr="//div[hasclass('col-lg-6')]" position="attributes">
+        <attribute name="class" add="o_cc5" remove="o_cc4" separator=" " />
+        <attribute name="style" add="text-align: center; !important" separator=" " />
```

Comment:
Across the whole commit: isn't `!important` supposed to be before the semi-colon ???